src/components/Header.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch, faShoppingCart } from '@fortawesome/free-solid-svg-icons'
+
+function Header() {
+    return (
+        <header className="wz-header">
+            <input type="checkbox" id="nav-check"></input>
+            <div className="nav-header">
+                <div className="nav-title">Furnituria</div>
+            </div>
+            <div className="nav-btn">
+                <label htmlFor="nav-check">
```

Comment:
Estos span de momento no tienen utilidad o motivo para estar? quiza podrias borrarlos

Reply:
These spans have the capability for making the hamburger emblem.
But right now I'm looking that FontAwesome has the bars icon, so I'll replace it.

Reply:
Fixed !